connectors/slack.rb

```diff
@@ -0,0 +1,69 @@
+require 'slack'
+
+class SlackConnector < Linkbot::Connector
+  Linkbot::Connector.register('slack', self)
+
+  def initialize(options)
+    super(options)
+
+    Slack.configure do |config|
+      config.token = @options["token"]
+    end
+
+    puts "Creating slack client"
+    @client = Slack.client
+
+    puts "Connection realtime client"
+    @rtc = @client.realtime
+
+    puts "Client connected"
+    @rtc.on :message do |data|
+      if data["channel"]
+        process_message(Time.at(data["ts"].to_i),data["user"],data["text"],{:room => data["channel"]})
+      end
+    end
+
+    update_users(@client.post("rtm.start")["users"])
+  end
+
+  def start
+    @rtc.start
+  end
```

Comment:
Moving the `@rtc.start` into this `start` method got the Slack connector working.

I think the calls to `listen` in the other connectors' `initialize` will need to move into `start` methods for each connector because of [Reasons](https://github.com/markolson/linkbot/blob/mark-refactor/lib/linkbot.rb#L68).